### PR TITLE
[BZ 1300727] Fix validateServerAttributes() for fallback case

### PR DESCRIPTION
**Pull request by burmanm**

### CODE REVIEW

**Comment by tsegismont** [comment]
For some reason, the build failed in core-domain module...

**Comment by tsegismont** [comment]
retest this please

File: modules/plugins/jboss-as-7/src/test/java/org/rhq/modules/plugins/jbossas7/itest/AbstractServerComponentTest.java [hunk 1]
```diff
@@ -107,8 +107,10 @@ public void testServerAttributeValidation() throws Exception {
             + "start() method due to invalid baseDir.");
 
         // Change the expectedRuntimeProductName property
+        boolean exceptException = true;
```
Comment (tsegismont) [inline modules/plugins/jboss-as-7/src/test/java/org/rhq/modules/plugins/jbossas7/itest/AbstractServerComponentTest.java]: Typo?

**Comment by tsegismont** [comment]
@burmanm could you sum up what is the intention? There is a lot comments in the related BZs... My understanding is that we want non identifiable EAP6-based products to be discovered as raw EAP6 servers, but it doesn't look like it is what this PR does. Thanks

File: modules/plugins/jboss-as-7/src/test/java/org/rhq/modules/plugins/jbossas7/itest/AbstractServerComponentTest.java (hunk 1 above)
Comment (burmanm) [inline modules/plugins/jboss-as-7/src/test/java/org/rhq/modules/plugins/jbossas7/itest/AbstractServerComponentTest.java]: Yes, a typo. Shouldn't matter though too much.

**Comment by burmanm** [comment]
This isn't the patch that's doing that, this is just fixing what was wrong in the original commit to the release/jon3.3.x (one extra line and incorrectly behaving validateServerAttributes()), see comment 8 on the ticket.

File: modules/plugins/jboss-as-7/src/test/java/org/rhq/modules/plugins/jbossas7/itest/AbstractServerComponentTest.java (hunk 1 above)
Comment (tsegismont) [inline modules/plugins/jboss-as-7/src/test/java/org/rhq/modules/plugins/jbossas7/itest/AbstractServerComponentTest.java]: Not too much indeed. But without the typo you can  understand at first glance :)

**Comment by tsegismont** [comment]
Le 11/03/2016 21:53, Michael Burman a écrit :
> This isn't the patch that's doing that, this is just fixing what was
> wrong in the original commit to the release/jon3.3.x (one extra line and
> incorrectly behaving validateServerAttributes()), see comment 8 on the
> ticket.
>

Got it now. Thanks

**Comment by tsegismont** [comment]
LGTM. Do you know what happened to the build? It failed twice while building core-domain... Maybe a PR has been merged before which wasn't quite right.

**Pull request merged**